src/ServerTrait.php

```diff
@@ -44,4 +44,13 @@ private function pause($message): void
             }
         }
     }
+
+    private function takeScreenshot(string $type, string $test): void
+    {
+        if (!($_SERVER['PANTHER_SCREENSHOT_DIR'] ?? false) || !$client = PantherTestCase::getPrimaryPantherClient()) {
+            return;
+        }
+
+        $client->takeScreenshot(sprintf('%s_%s.png', $type, strtr($test, ['\\' => '-', ':' => '_'])));
```

Comment:
I suggest prepending the path by the current date and time.

Reply:
I thought of this too but decided against this as it could make your screenshot directory huge when running the test suite many times with many failures. I figured letting them override the old files would be best.

That being said, I'll prepend the date/time if you'd like. Just wanted to give my reasoning for not doing it.

Reply:
I fear it can make things hard to debug in case of override. I prefer to save everything and clean after than loosing some maybe important debugging data.

Reply:
See 6a23c013bf08297d5b235afe5dbe35f6026818af, let me know if you'd like a different format.